libs/api-client/src/api-client.ts

```diff
@@ -280,26 +280,28 @@ export class GFW_API_CLASS {
         if (this.debug) {
           console.log(`GFWAPI: Fetching URL: ${url}`)
         }
+        const finalHeaders = {
+          ...headers,
+          ...(requestType === 'json' && { 'Content-Type': 'application/json' }),
+          ...(local && {
+            'x-gateway-url': API_GATEWAY,
+            user: JSON.stringify({
+              id: process.env.REACT_APP_LOCAL_API_USER_ID,
+              type: process.env.REACT_APP_LOCAL_API_USER_TYPE,
+              email: process.env.REACT_APP_LOCAL_API_USER_EMAIL,
+            }),
+          }),
+          Authorization: `Bearer ${this.getToken()}`,
+        }
         const fetchUrl = isUrlAbsolute(url)
           ? url
           : this.baseUrl + (dataset ? `/datasets/${this.dataset}` : '') + url
         const data = await fetch(fetchUrl, {
+          redirect: 'follow',
```

Comment:
what is this? and why is it needed as a fixed param?

Reply:
I was having issues with the API and this was one intent to make it work, removing.